<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1068</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="currentLyric">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>910</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Welcome!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>960</y>
     <width>1171</width>
     <height>103</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>293</width>
        <height>86</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="prevBtn">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playerBtn">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextBtn">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modeBtn">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>18</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>461</width>
        <height>101</height>
       </size>
      </property>
      <widget class="QLabel" name="albumPic">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QSlider" name="processBar">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #bbb; /* 滑槽的边框颜色 */
    background: #ccc; /* 滑槽的背景颜色 */
    height: 8px; /* 滑槽的高度 */
    border-radius: 4px; /* 滑槽的圆角半径 */
}
 
QSlider::handle:horizontal {
    background: #FC3D49; /* 滑块的背景颜色 */
    border: 1px solid #FC3D49; /* 滑块的边框颜色 */
    width: 18px; /* 滑块的宽度 */
    height: 18px; /* 滑块的高度 */
    border-radius: 9px; /* 滑块的圆角半径，确保滑块是圆形 */
    margin: -6px 0; /* 滑块超出滑槽的边距 */
}
 
QSlider::sub-page:horizontal {
    /* 使用线性渐变实现已通过部分的渐变效果 */
    background: qlineargradient(
        spread: pad, /* 填充模式 */
        x1: 0, y1: 0, /* 起点位置 */
        x2: 1, y2: 0, /* 终点位置（水平渐变） */
        stop: 0 #FC3D49, /* 起始颜色 */
        stop: 1 #fc9c9e /* 结束颜色 */
    );
    /*border: 1px solid #FC3D49; /* 已通过部分的边框颜色 */
    height: 8px; /* 已通过部分的高度 */
    border-radius: 4px; /* 已通过部分的圆角半径 */
}
 
QSlider::add-page:horizontal {
    background: #ccc; /* 未通过部分的背景颜色 */
    border: 1px solid #bbb; /* 未通过部分的边框颜色 */
    height: 8px; /* 未通过部分的高度 */
    border-radius: 4px; /* 未通过部分的圆角半径 */
}
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="songName">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>180</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI Light</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>我怀念的 - 孙燕姿</string>
       </property>
      </widget>
      <widget class="QLabel" name="currentTime">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>50</y>
         <width>61</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>03:51/</string>
       </property>
      </widget>
      <widget class="QLabel" name="totalTime">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>50</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>04:50</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>18</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>181</width>
        <height>80</height>
       </size>
      </property>
      <widget class="QSlider" name="soundBar">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #bbb; /* 滑槽的边框颜色 */
    background: #ccc; /* 滑槽的背景颜色 */
    height: 8px; /* 滑槽的高度 */
    border-radius: 4px; /* 滑槽的圆角半径 */
}
 
QSlider::handle:horizontal {
    background: #FC3D49; /* 滑块的背景颜色 */
    border: 1px solid #FC3D49; /* 滑块的边框颜色 */
    width: 18px; /* 滑块的宽度 */
    height: 18px; /* 滑块的高度 */
    border-radius: 9px; /* 滑块的圆角半径，确保滑块是圆形 */
    margin: -6px 0; /* 滑块超出滑槽的边距 */
}
 
QSlider::sub-page:horizontal {
    /* 使用线性渐变实现已通过部分的渐变效果 */
    background: qlineargradient(
        spread: pad, /* 填充模式 */
        x1: 0, y1: 0, /* 起点位置 */
        x2: 1, y2: 0, /* 终点位置（水平渐变） */
        stop: 0 #FC3D49, /* 起始颜色 */
        stop: 1 #fc9c9e /* 结束颜色 */
    );
    /*border: 1px solid #FC3D49; /* 已通过部分的边框颜色 */
    height: 8px; /* 已通过部分的高度 */
    border-radius: 4px; /* 已通过部分的圆角半径 */
}
 
QSlider::add-page:horizontal {
    background: #ccc; /* 未通过部分的背景颜色 */
    border: 1px solid #bbb; /* 未通过部分的边框颜色 */
    height: 8px; /* 未通过部分的高度 */
    border-radius: 4px; /* 未通过部分的圆角半径 */
}</string>
       </property>
       <property name="value">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="soundBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>1141</width>
     <height>721</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_music">
    <widget class="RotateWidget" name="PostWidget" native="true">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>230</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>190</y>
       <width>276</width>
       <height>370</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lyricLabel_1">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lyricLabel_2">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lyricLabel_3">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Welcome!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lyricLabel_4">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lyricLabel_5">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lyricLabel_6">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lyricLabel_7">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QListWidget" name="musicList">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>110</y>
       <width>241</width>
       <height>521</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>150</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 设置 QListWidget 的整体外观，圆角和背景颜色 */
QListWidget {
    border: 1px solid #ccc;
    border-radius: 10px; /* 圆角 */
    background-color: #f9f9f9; /* 背景色 */
    padding: 5px;
	outline:0px;
}

/* 设置 QListWidget 中每个选项的默认外观 */
QListWidget::item {
    border: 1px solid transparent;
    border-radius: 5px; /* 圆角 */
    background-color: #ffffff; /* 默认选项背景色 */
    padding: 8px;
    margin: 3px; /* 每个选项的间距 */
    color: #333; /* 字体颜色 */
    font-size: 14px;
}

/* 鼠标悬停时的效果 */
QListWidget::item:hover {
    background-color: #ffe6e8; /* 悬停背景色 */
    /*border: 1px solid #91d5ff; /* 悬停时的边框颜色 */
}

/* 选中状态的效果 */
QListWidget::item:selected {
    background-color: #FC3D49; /* 选中背景色 */
    /*border: 1px solid #69c0ff; /* 选中边框颜色 */
    color: #ffffff; /* 选中文字颜色 */
}


/* 自定义滑动条的样式 */
QScrollBar:vertical {
    border: none; /* 移除边框 */
    background: #f0f0f0; /* 滑动条背景色 */
    width: 8px; /* 设置滑动条宽度 */
    margin: 0px 0px 0px 0px; /* 滑动条上下的边距 */
    border-radius: 4px; /* 圆角效果 */
}

QScrollBar::handle:vertical {
    background: #bfbfbf; /* 滑块颜色 */
    border-radius: 4px; /* 圆角效果 */
    min-height: 20px; /* 滑块的最小高度 */
}

QScrollBar::handle:vertical:hover {
    background: #a6a6a6; /* 悬停时滑块的颜色 */
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    background: none; /* 隐藏滑动条上下的按钮 */
    height: 0px; /* 移除按钮高度 */
    width: 0px;
}

/* 防止滑块超出范围 */
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}

</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_home"/>
   <widget class="QWidget" name="page_love"/>
   <widget class="QWidget" name="page_download"/>
   <widget class="QWidget" name="page_local"/>
  </widget>
  <widget class="QPushButton" name="pushButton_page_music">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>190</y>
     <width>222</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid transparent;
    border-radius: 10px;
	text-align: left; padding-left: 10px;
    color: #757c8a;
	font-size: 19px;
}

QPushButton:hover {
    background-color: rgba(169, 169, 169, 0.5);  /* 灰色提示 */
}
</string>
   </property>
   <property name="text">
    <string> 音乐</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/page_play.png</normaloff>:/img/page_play.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>34</width>
     <height>34</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_page_home">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>141</y>
     <width>222</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid transparent;
    border-radius: 10px;
    /*padding: 10px 15px;*/
	color: #757c8a;
	font-size: 19px;
	text-align: left; padding-left: 10px;
}

QPushButton:hover {
    background-color: rgba(169, 169, 169, 0.5);  /* 灰色提示 */
}

</string>
   </property>
   <property name="text">
    <string> 推荐</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/page_home.png</normaloff>:/img/page_home.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>34</width>
     <height>34</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_page_love">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>507</y>
     <width>222</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid transparent;
    border-radius: 10px;
   	text-align: left; padding-left: 10px;
    color: #757c8a;
	font-size: 19px;
}

QPushButton:hover {
    background-color: rgba(169, 169, 169, 0.5);  /* 灰色提示 */
}
</string>
   </property>
   <property name="text">
    <string> 我喜欢的音乐</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/list_love.png</normaloff>:/img/list_love.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_page_download">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>727</y>
     <width>222</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid transparent;
    border-radius: 10px;
    text-align: left; padding-left: 10px;
    color: #757c8a;
	font-size: 19px;
}

QPushButton:hover {
    background-color: rgba(169, 169, 169, 0.5);  /* 灰色提示 */
}
</string>
   </property>
   <property name="text">
    <string> 下载管理</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/list_download.png</normaloff>:/img/list_download.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>34</width>
     <height>34</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_page_local">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>779</y>
     <width>222</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: 2px solid transparent;
    border-radius: 10px;
    text-align: left; padding-left: 10px;
    color: #757c8a;
	font-size: 19px;
}

QPushButton:hover {
    background-color: rgba(169, 169, 169, 0.5);  /* 灰色提示 */
}
</string>
   </property>
   <property name="text">
    <string> 本地音乐</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/list_local.png</normaloff>:/img/list_local.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>34</width>
     <height>34</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>30</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_playlist">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>990</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/icon_playlist.png</normaloff>:/img/icon_playlist.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RotateWidget</class>
   <extends>QWidget</extends>
   <header location="global">rotatewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
